spec/components/support_interface/application_choice_component_spec.rb

```diff
@@ -64,6 +64,30 @@
       expect(page).not_to have_selector '.app-summary-card__actions a'
       expect(page).not_to have_text 'Change offered course'
     end
+
+    it 'renders a link to change conditions' do
+      result = render_inline(described_class.new(accepted_choice))
+
+      expect(result.css('.app-summary-card .govuk-summary-list__actions a').first.text.squish).to eq 'Change conditions'
+    end
+
+    context 'when one of the conditions is met' do
+      let(:conditions) { [build(:offer_condition, status: 'met'), build(:offer_condition)] }
+      let(:accepted_choice) do
+        create(
+          :application_choice,
+          :with_completed_application_form,
+          :with_accepted_offer,
+          offer: build(:offer, conditions: conditions),
+        )
+      end
+
+      it 'does not render a link to the change the offered course choice  when the`change_offered_course` flag is not active' do
+        result = render_inline(described_class.new(accepted_choice))
+
+        expect(result.css('.app-summary-card .govuk-summary-list__actions a').text.squish).not_to include 'Change conditions'
```

Comment:
Isn't it the case that `result.css('.app-summary-card .govuk-summary-list__actions a')` won't exist for this context as no anchor tag will be rendered?

Reply:
Yes this is trying to test that there is no change link for conditions rendered.

Since there are other change links which have the same selector on the page, I just checked that there isn't one for conditions. Open to suggestions for making that check a bit more explicit

Reply:
:+1: Ah I see, was forgetting about other change links being present.